### PR TITLE
Enable reduce_sum operator for Ngraph

### PR DESCRIPTION
**Pull request by pawelpiotrowicz**
Enable the reduce_sum operator for the nGraph bridge. BERT model uses it.

### CODE REVIEW

File: paddle/fluid/operators/ngraph/ops/reduce_sum_op.h [hunk 1]
```diff
@@ -0,0 +1,161 @@
+/*Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```
Comment (mozga-intel) [inline paddle/fluid/operators/ngraph/ops/reduce_sum_op.h]: +1

File: python/paddle/fluid/tests/unittests/ngraph/test_reduce_ngraph_op.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```
Comment (mozga-intel) [inline python/paddle/fluid/tests/unittests/ngraph/test_reduce_ngraph_op.py]: +1

File: paddle/fluid/operators/ngraph/ops/reduce_sum_op.h (hunk 1 above)
Comment (pawelpiotrowicz) [reply paddle/fluid/operators/ngraph/ops/reduce_sum_op.h]: done

File: python/paddle/fluid/tests/unittests/ngraph/test_reduce_ngraph_op.py (hunk 2 above)
Comment (pawelpiotrowicz) [reply python/paddle/fluid/tests/unittests/ngraph/test_reduce_ngraph_op.py]: done

**Comment by luotao1** [comment]
CLA has some problem several hours ago, you can close and reopen this PR again. Then CLA will be triggered successfully.

**Comment by luotao1** [comment]
Do you sign CLA before?

**Comment by luotao1** [comment]
How about create a new PR?

**Review by baojun-nervana — APPROVED** [review]
LGTM

**Review by tensor-tang — APPROVED** [review]
LGTM

**Pull request merged**